src/Monolog/Handler/SwiftMailerHandler.php

```diff
@@ -42,6 +43,8 @@ public function __construct(\Swift_Mailer $mailer, $message, $level = Logger::ER
     {
         parent::__construct($level, $bubble);
 
+        @trigger_error('The SwiftMailerHandler is deprecated. Use SymfonyMailerHandler instead.', E_USER_DEPRECATED);
```

Comment:
It's better to always mention the package+version that the deprecation is for since this makes it easier to understand aggregated deprecation reports that might be about various packages. (same in the annotations btw)

`since monolog/monolog v2.6`